docs/setup/upgrade/adopting-lts-1.2.0/upgrade-runbook-1.2.1.0.md

```
@@ -20,44 +17,68 @@ This document provides step-by-step instructions to help you:
+* Take backup of the required services (as listed below) for any restore:
   * Postgress
-    * Refer: [https://github.com/mosip/mosip-infra/tree/release-1.2.1.x/deployment/v3/external/postgres#db-export](https://github.com/mosip/mosip-infra/tree/release-1.2.1.x/deployment/v3/external/postgres#db-export)
-    * Make sure to backup all the schema along with roles , users and password.
-  * SoftHSM (this applies only to sandbox or testing environment upgrades and must not be used in production. For RealHSM, the upgrade process should be guided by the HSM administrator or the vendor.)
-    * Refer: [https://github.com/mosip/mosip-infra/tree/release-1.2.1.x/deployment/v3/external/hsm/softhsm#backup-softhsm](https://github.com/mosip/mosip-infra/tree/release-1.2.1.x/deployment/v3/external/hsm/softhsm#backup-softhsm)
-  * minio
-    * Refer: [https://github.com/mosip/mosip-infra/tree/release-1.2.1.x/deployment/v3/external/object-store/minio](https://github.com/mosip/mosip-infra/tree/release-1.2.1.x/deployment/v3/external/object-store/minio)
+    * [Refer](https://github.com/mosip/mosip-infra/tree/release-1.2.1.x/deployment/v3/external/postgres#db-export)
+      * Make sure to backup all the schema along with roles , users and password.
+  * [SoftHSM](https://github.com/mosip/mosip-infra/tree/release-1.2.1.x/deployment/v3/external/hsm/softhsm#backup-softhsm) (this applies only to sandbox or testing environment upgrades and must not be used in production. For RealHSM, the upgrade process should be guided by the HSM administrator or the vendor.)
+  * [minio]((https://github.com/mosip/mosip-infra/tree/release-1.2.1.x/deployment/v3/external/object-store/minio)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)